test-resources-plugin/src/main/java/io/micronaut/gradle/testresources/MicronautTestResourcesPlugin.java

```diff
@@ -175,7 +172,7 @@ private static void configureServerConnection(TaskProvider<StartTestResourcesSer
                 return testResourcesSourceSet.getRuntimeClasspath();
             }
             return Collections.emptyList();
-        }));
+        })).withPropertyName("testResourcesSourceSet.RuntimeClasspath").withPathSensitivity(PathSensitivity.RELATIVE);
```

Comment:
should this be relative path sensitivity or runtime classpath?

Reply:
It's set as task inputs, so there's no way to set it as runtimeClasspath.